2022-01-18 13:50:33,145 INFO    MainThread:589752 [wandb_setup.py:_flush():71] setting env: {}
2022-01-18 13:50:33,145 INFO    MainThread:589752 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 13:50:33,145 INFO    MainThread:589752 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 13:50:33,145 INFO    MainThread:589752 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220118_135033-18qe3bvm/logs/debug.log
2022-01-18 13:50:33,145 INFO    MainThread:589752 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220118_135033-18qe3bvm/logs/debug-internal.log
2022-01-18 13:50:33,146 INFO    MainThread:589752 [wandb_init.py:init():404] calling init triggers
2022-01-18 13:50:33,146 INFO    MainThread:589752 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-18 13:50:33,146 INFO    MainThread:589752 [wandb_init.py:init():460] starting backend
2022-01-18 13:50:33,146 INFO    MainThread:589752 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-18 13:50:33,160 INFO    MainThread:589752 [backend.py:ensure_launched():216] starting backend process...
2022-01-18 13:50:33,164 INFO    MainThread:589752 [backend.py:ensure_launched():221] started backend process with pid: 589807
2022-01-18 13:50:33,165 INFO    MainThread:589752 [wandb_init.py:init():469] backend started and connected
2022-01-18 13:50:33,167 INFO    MainThread:589752 [wandb_init.py:init():533] updated telemetry
2022-01-18 13:50:33,176 INFO    MainThread:589752 [wandb_init.py:init():563] communicating current version
2022-01-18 13:50:33,533 INFO    MainThread:589752 [wandb_init.py:init():568] got version response 
2022-01-18 13:50:33,534 INFO    MainThread:589752 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-18 13:50:33,727 INFO    MainThread:589752 [wandb_init.py:init():606] starting run threads in backend
2022-01-18 13:50:33,798 INFO    MainThread:589752 [wandb_run.py:_console_start():1810] atexit reg
2022-01-18 13:50:33,798 INFO    MainThread:589752 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-18 13:50:33,799 INFO    MainThread:589752 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-18 13:50:33,800 INFO    MainThread:589752 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-18 13:50:33,800 INFO    MainThread:589752 [wandb_init.py:init():633] run started, returning control to user process
2022-01-18 13:50:35,253 INFO    MainThread:589752 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-18 13:50:35,254 INFO    MainThread:589752 [wandb_run.py:_restore():1752] restore
2022-01-18 13:50:37,439 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 777
}

2022-01-18 13:50:37,720 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 777
}

2022-01-18 13:50:37,822 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 7901
}

2022-01-18 13:50:37,924 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}

2022-01-18 13:50:38,027 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}

2022-01-18 13:50:38,131 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}

2022-01-18 13:50:38,235 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}

2022-01-18 13:50:38,339 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}

2022-01-18 13:50:38,705 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}

2022-01-18 13:50:39,096 INFO    MainThread:589752 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7901
  total_bytes: 7901
}
local_info {
}

2022-01-18 13:50:40,150 INFO    MainThread:589752 [wandb_run.py:_append_files():2180] logging synced files
